src/org/labkey/test/pages/pipeline/PipelineStatusDetailsPage.java

```diff
@@ -183,10 +183,10 @@ public PipelineStatusDetailsPage waitForComplete(int wait)
 
         if (expectRedirect)
         {
-            // The click does nothing, but we want to wait for the redirect before continuing
-            log("Process done, clicking and waiting for redirect");
-            clickAndWait(elementCache().statusText);
-            log("Redirected on success as expected: " + getDriver().getCurrentUrl());
+            doAndWaitForPageToLoad(() -> {
+                log("Process done, clicking and waiting for redirect");
+                log("Redirected on success as expected: " + getDriver().getCurrentUrl());
+            });
```

Comment:
Should log the success message after redirecting.
```suggestion
            });
            log("Redirected on success as expected: " + getDriver().getCurrentUrl());
```